containers/jvm/clipper-spark/src/test/scala/ai/clipper/spark/ClipperTest.scala

```diff
@@ -0,0 +1,346 @@
+package ai.clipper.spark
```

Comment:
This file is a duplicate of `containers/jvm/clipper-spark/src/test/scala/ai/clipper/spark/Clipper.scala`